viewing_party/party.py

```diff
@@ -1,23 +1,183 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+def create_movie(movie_title, genre, rating):
+    if movie_title == None or genre == None or rating == None:

```

Comment:
Nice guard clause!

It would be more Pythonic to write this check as:
```suggestion
    if not movie_title or not genre or not rating:
```